### PR TITLE
Increasing price of mining RIGs in vendor

### PR DESCRIPTION
**Pull request by Sindorman**
as title says this PR increases price of mining RIG in vending machine.

Reason: RIGs are really powerful, even mining. Since RND can produce them with basic materials, this will increase relationship between mining and RND.

### CODE REVIEW

**Comment by Sindorman** [comment]
!review

File: html/changelogs/Sindorman-mining.yml [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+# Optional: Remove this file after generating master changelog.  Useful for PR changelogs that won't get used again.
+delete-after: True
+
+# Any changes you've made.  See valid prefix list above.
+# INDENT WITH TWO SPACES.  NOT TABS.  SPACES.
+# SCREW THIS UP AND IT WON'T WORK.
+# Also, all entries are changed into a single [] after a master changelog generation. Just remove the brackets when you add new entries.
+# Please surround your changes in  double quotes ("), as certain characters otherwise screws up compiling. The quotes will not show up in the changelog.
+changes: 
+  - rscdel: "Removed Industrail RIG from mining vending machine."
```
Comment (Karolis2011) [inline html/changelogs/Sindorman-mining.yml]: Industrail? Industrial?

File: code/modules/mining/machine_vending.dm [hunk 2]
```diff
@@ -44,8 +44,8 @@ var/global/list/minevendor_list = list( //keep in order of price
 	new /datum/data/mining_equipment("Ore Magnet",					/obj/item/weapon/oremagnet,									10,					600),
 	new /datum/data/mining_equipment("Minecart",					/obj/vehicle/train/cargo/trolley/mining,					-1,					600,	1),
 	new /datum/data/mining_equipment("Resonator",					/obj/item/weapon/resonator,									10,					700),
-	new /datum/data/mining_equipment("Mining RIG",					/obj/item/weapon/rig/industrial,							5,					750),
 	new /datum/data/mining_equipment("Jaunter",						/obj/item/device/wormhole_jaunter,							20,					750),
+	new /datum/data/mining_equipment("Mining RIG",					/obj/item/weapon/rig/industrial,							2,					1000),
```
Comment (Karolis2011) [inline code/modules/mining/machine_vending.dm]: Maybe put it back on same line?

File: html/changelogs/Sindorman-mining.yml [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+# Optional: Remove this file after generating master changelog.  Useful for PR changelogs that won't get used again.
+delete-after: True
+
+# Any changes you've made.  See valid prefix list above.
+# INDENT WITH TWO SPACES.  NOT TABS.  SPACES.
+# SCREW THIS UP AND IT WON'T WORK.
+# Also, all entries are changed into a single [] after a master changelog generation. Just remove the brackets when you add new entries.
+# Please surround your changes in  double quotes ("), as certain characters otherwise screws up compiling. The quotes will not show up in the changelog.
+changes: 
+  - rscdel: "Removed Industrial RIG from mining vending machine."
```
Comment (Karolis2011) [inline html/changelogs/Sindorman-mining.yml]: Changelog is now outdated.

File: code/modules/mining/machine_vending.dm (hunk 2 above)
Comment (Sindorman) [reply code/modules/mining/machine_vending.dm]: Does not matter really

**Comment by Geevies** [comment]
This is a bit much. The mining RIG isn't exceptionally powerful, and ever since the module locking thing, where RIGs of a certain type could only take their class of modules, RIGs have been fairly balanced.
Robotics can produce them, yes, but there aren't always roboticists playing on a round, leaving a pretty important part of mining's progression out of reach.
Considering the amount of miners there can be a round, especially with cryoing and such, I'm not so sure whether this a good change.

**Comment by sheeplets** [comment]
A feedback thread would be appreciated considering how badly this would cripple the entire mining department.

**Comment by Sindorman** [comment]
@Arrow768 @skull132 Should we make feedback thread for this?

**Comment by KingOfThePing** [comment]
I don't see how this cripples the mining department. There are ample other methods of getting ores, namely KAs, ***drills***, etc. Good change, I, personnally, am very tired of seeing every miner and their mother with a deployed RIG running around all round.

**Comment by Arrow768** [comment]
I dont see how this is going to "cripple" the mining department.
At this point it´s just a price change, but I would also be fine with a price and amount change.
Afaik we have 4 slots so 2 rigs would be sufficient (Rigs go to the fastest miners and the slower ones will just have to ask research)

**Pull request merged**